python/docs/source/user_guide/sql/python_udtf.rst

```diff
@@ -285,10 +327,39 @@ To implement a Python UDTF, you first need to define a class implementing the me
             """
             ...
 
+Emitting output rows
+--------------------
+
+The return type of the UDTF defines the schema of the table it outputs. It must be either a
+``StructType``, for example ``StructType().add("c1", StringType())``, or a DDL string representing a
+struct type, for example ``c1: string``. The `eval` and `terminate` methods then emit zero or more
+output rows conforming to this schema by yielding tuples, lists, or pyspark.sql.Row objects. For
+example:
+
+```
+def eval(self, x, y, z):
+    # Here we return a row by providing a tuple of three elements.
```

Comment:
This section is very helpful! Could we move this comment outside of the code block for the following examples? Like

Return a row by providing a tuple of three elements.
```
def eval(...)
```
...